### DISCUSSION TITLE
Invalid request: missing required property `email`.

### DISCUSSION THREAD

**Discussion by aranajhonny** [vercel/community#7417] [Account Issue] [answered]
### Question

![image](https://github.com/vercel/community/assets/9091881/c884747c-a491-415c-b68d-752f579a6630)

**Comment by pawlean** [comment, marked as answer]
Hi @aranajhonny!

I can confirm there was an incident yesterday that caused this. Are you able to login now?

**Reply by aranajhonny** [reply]
Successful

[a comment by an automated account is not shown]
